tasks/main.yml

```diff
@@ -7,6 +7,11 @@
 - name: Include OS variables
   include_vars: "{{ ansible_os_family }}.yml"
 
+- name: Set nomad_servers to empty list rather than the default macro if we are using consul
```

Comment:
It's needed for me as I get a jinja evaluation failure, ansible evaluates it anyway, which I hate.